config/helm/patches/crds/all.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: core
+  labels:
+    clusterctl.cluster.x-k8s.io: ""
+    clusterctl.cluster.x-k8s.io/move: ""
```

```suggestion
    clusterctl.cluster.x-k8s.io/move: ""
```

Please add a newline at EOF. Thank you!